modules/k8s_resource/tf_module/main.tf

```diff
@@ -0,0 +1,13 @@
+provider "kubernetes" {
+  config_path    = var.kubeconfig
+  config_context = var.kubecontext
+}
+
+resource "kubernetes_manifest" "manifest" {
+  manifest = yamldecode(file(var.file_path))
+  timeouts {
+    update = "2m"
+    create = "5m"
+    delete = "2m"
```

Comment:
2m, could be a bit low?
I wonder if we can use the same value for these 3 and support an optional module variable `timeout_min`.
just in case someone needs a higher timeout they could set it up from config file, and we don't need to change opta

Reply:
5m for each ok?
About the other variable, We could add it when a customer runs into 5 mins being too short, but its probably not needed right now (for example, when using kubectl we dont specify timeouts).

Reply:
yes 5min for all 👍